purchases/src/main/kotlin/com/revenuecat/purchases/common/AppConfig.kt

```diff
@@ -35,6 +35,9 @@ internal class AppConfig(
     } ?: URL("https://api.revenuecat.com/")
     val diagnosticsURL = URL("https://api-diagnostics.revenuecat.com/")
 
+    val playStoreVersionName = context.playStoreVersionName
+    val playServicesVersionName = context.playServicesVersionName
```

Comment:
I dont particularly like eagerly retrieving these, but we are not storing the Context instance from the constructor so it's either doing it eagerly here or storing the Context and switching these to be lazy getters :/

Reply:
Hmm do you have any concerns with eagerly retrieving these? In general we should try to store the context whenever possible I think. It should be mostly fine if the user gives us the application context (which they should), but if they gave us an activity context it could lead to leaks. So I think it's better this way. Happy to talk more if you want!

Reply:
Only concern is doing more work than is strictly needed since this data will only be used on apps with diagnostics enabled (for now).